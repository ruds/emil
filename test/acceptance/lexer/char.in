#"a" #"b" #"ðŸ˜ƒ" #"Ã€"
#"'" #"\"" #"\\" #"\$" #"\a" #"\b" #"\f" #"\n" #"\r" #"\t" #"\v"
#"\^@" #"\^A" #"\^^" #"\^_"
#"\u1234" #"\u263A" #"\u00c0"
#"\U01f603" #"\U01F60D"
#"\c"
(*xx*)
#"\^a"
(*xx*)
#"aa"
(*xx*)
#"\udc00"
(*xx*)
#"\u01f603"
(*xx*)
#"\U000000c0"
(*xx*)
#"\U220000"
(*xx*)
#"\u-0020"
(*xx*)
