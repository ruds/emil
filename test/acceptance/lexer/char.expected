0001 CHAR 000061 #"a"
0001 CHAR 000062 #"b"
0001 CHAR 01F603 #"ðŸ˜ƒ"
0001 CHAR 0000C0 #"Ã€"
0002 CHAR 000027 #"'"
0002 CHAR 000022 #"\""
0002 CHAR 00005C #"\\"
0002 CHAR 000024 #"\$"
0002 CHAR 000007 #"\a"
0002 CHAR 000008 #"\b"
0002 CHAR 00000C #"\f"
0002 CHAR 00000A #"\n"
0002 CHAR 00000D #"\r"
0002 CHAR 000009 #"\t"
0002 CHAR 00000B #"\v"
0003 CHAR 000000 #"\^@"
0003 CHAR 000001 #"\^A"
0003 CHAR 00001E #"\^^"
0003 CHAR 00001F #"\^_"
0004 CHAR 001234 #"\u1234"
0004 CHAR 00263A #"\u263A"
0004 CHAR 0000C0 #"\u00c0"
0005 CHAR 01F603 #"\U01f603"
0005 CHAR 01F60D #"\U01F60D"
0006 ERROR
0008 ERROR
0010 ERROR
0012 ERROR
0014 ERROR
0016 ERROR
0018 ERROR
0020 ERROR
0022 END
